README.md

```diff
@@ -96,6 +96,7 @@ Follow the usual GitHub workflow of forking the repository on GitHub and then cl
 5. Start your local Hugo server:
 
     ```bash
+    cd ../.. #You should be in the root directory of the repository
```

Comment:
Actually, if you've run the above command (step 4) exactly like it's written (with parentheses), you don't need this `cd ../..`.